src/log-analytics-solution/azext_log_analytics_solution/_help.py

```diff
@@ -0,0 +1,71 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['monitor log-analytics solution'] = """
+    type: group
+    short-summary: Commands to manage monitor log-analytics solution.
+"""
+
+helps['monitor log-analytics solution create'] = """
+    type: command
+    short-summary: Create a log-analytics solution.
+    examples:
+      - name: Create a log-analytics solution
+        text: |-
+               az monitor log-analytics solution create --resource-group MyResourceGroup \\
+               --name MySolutionGroup --location "East US" --tags key=value \\
+               --plan-name PlanName --plan-publisher PlanPublisher --plan-product PlanProduct \\
```

Comment:
Could the example be more specific for some arguments such a publisher, product?